schema/deploy/shipping/views.sql

```diff
@@ -844,4 +844,95 @@ comment on view shipping.hcov19_observation_v1 is
   'Custom view of hCoV-19 samples with presence-absence results and best available encounter data';
 
 
+create or replace view shipping.scan_return_results_v1 as
+
+    with status_codes as (
+        -- Create status codes as required by Lab Med's results portal
+        with x(status_code, status) as (
+            values
+            ('DET', 'positive result, colored red + bold'),
+            ('INCON', 'inconclusive, colored orange; treated as positive with respect to information given'),
+            ('NDET', 'not detected/negative, colored blue'),
+            ('pending-other', null),
+            ('pending-lab', 'We are working on it!'),
+            ('cancelled', null),
+            ('notReceived', 'This is our code we made up')
+        )
+        select * from x
+    ),
+
+    results as (
+        select distinct on (barcode, lineage)
+            sample_id,
+            barcode as qrcode,
+            encountered as collect_ts,
+            case
+                when sample_id is null then 'notReceived'  -- todo new code
+                when sample_id is not null and present is null then 'pending-lab'
+                when target.identifier is not null and present is null then 'INCON'
```

Comment:
I think with this current logic it won't ever reach 'INCON'.

Reply:
Thanks for pointing that out. I wasn't sure what classified as an inconclusive result. Do you have any insight here? This may not be determinable until we see the new Samplify target data.

Reply:
I think you just need to flip the order of the two `when...` statements? But yeah, also not entirely sure how we are modeling inconclusive results within our database yet...

Reply:
I updated it to make it simpler: `if presence_absence_id is not null and present is null then 'inconclusive'`